import-scripts/transpose_cna_py3.py

```diff
@@ -0,0 +1,25 @@
+import argparse
```

Comment:
let's add some comments here about overall usage of the script - I know it's in the description but maybe on the top about use case and restrictions on passed in files

also can we add some comments about pandas functions -- think as a whole we're not too familiar so hard to read the code. `eg. df.T`

Reply:
done